resources/views/components/page-headers/transaction/icon-type.blade.php

```diff
@@ -1,4 +1,4 @@
-<x-general.entity-header-item :title="trans('pages.transaction.transaction_type')" icon="app-transactions.{{ $model->iconType() }}">
+<x-general.entity-header-item :title="trans('pages.transaction.transaction_type')" icon="app-transactions.{{ $model->iconType() }}" :wrapper-class="$wrapperClass ?? ''">
```

Comment:
might be worth splitting this onto separate lines as it's quite long

```suggestion
<x-general.entity-header-item
    :title="trans('pages.transaction.transaction_type')"
    icon="app-transactions.{{ $model->iconType() }}"
    :wrapper-class="$wrapperClass ?? ''"
>
```